calendar-app/src/api/saveAndEditEvent.ts

```diff
@@ -0,0 +1,87 @@
+import { StoredEvent, RequestResult } from "../utils/types";
+import { SERVER_URL } from "../pages/calendar/MainContent/consts";
+import apiRequest from "./sendAPIRequest";
+import { addEvent, updateEvent } from "../features/eventDisplay";
+import { store } from "../store";
+
+async function saveEvent(
+  form: StoredEvent,
+  id?: number
+): Promise<RequestResult> {
+  const isTitleEntered = !form.title || form.title.trim() === "";
```

Comment:
1. Either code or constant name should be inverted - currently it returns true when title is **not** entered
2. nit pick, the expression can be simplified:

```
const isTitleEntered = !!form.title?.trim()
```